token/program-2022/src/instruction.rs

```diff
@@ -692,11 +836,45 @@ impl AuthorityType {
             1 => Ok(AuthorityType::FreezeAccount),
             2 => Ok(AuthorityType::AccountOwner),
             3 => Ok(AuthorityType::CloseAccount),
+            4 => Ok(AuthorityType::FeeConfig),
+            5 => Ok(AuthorityType::FeeWithdraw),
             _ => Err(TokenError::InvalidInstruction.into()),
         }
     }
 }
 
+/// Specifies the authority type for SetAuthority instructions
+#[derive(Clone, Debug, PartialEq)]
+pub enum InitializeMintMixinParameters {
+    /// Additional padding required, in case the size of the account is the same
+    /// as a multisig
+    None {
+        padding: u64,
+    },
```

Comment:
I think just the tag name itself should be sufficient padding
```suggestion
    None,
```